code/lib/core-common/src/presets.ts

```diff
@@ -255,15 +255,13 @@ export async function loadPreset(
 
       if (
         storybookOptions.isCritical !== true &&
-        storybookOptions.build?.test?.removeNonFastAddons
+        (storybookOptions.build?.test?.disabledAddons?.length || 0) > 0
       ) {
         filter = (i: PresetConfig) => {
           // @ts-expect-error (Converted from ts-ignore)
           const name = i.name ? i.name : i;
 
-          return (
-            !name.includes('@storybook/addon-docs') && !name.includes('@storybook/addon-coverage')
-          );
+          return !!storybookOptions.build?.test?.disabledAddons?.find((n) => name.includes(n));
```

Comment:
@ndelangen I think the logic here might be backwards.  Instead of filtering out disabled addons, it _only allows_ disabled addons, since there is a double bang.  I found that my storybook started breaking in alpha.6 unless I provided an explicit `disabledAddons: []`.  Even just `disabledAddons: ['foo']` broke it, which I think is because it causes this code to execute.

Reply:
😱 I'll fix it